Encode.pm

```diff
@@ -499,30 +499,32 @@ followed by C<encode> as follows:
 
   $octets = encode_utf8($string);
 
+B<WARNING>: L</UTF-8 vs. utf8 vs. UTF8|This function can produce invalid UTF-8!>
+Do not use it for data exchange.
+Unless you want Perl’s older "lax" mode, prefer
+C<$octets = encode("UTF-8", $string)>.
+
 Equivalent to C<$octets = encode("utf8", $string)>.  The characters in
 $string are encoded in Perl's internal format, and the result is returned
 as a sequence of octets.  Because all possible characters in Perl have a
 (loose, not strict) utf8 representation, this function cannot fail.
 
-B<WARNING>: do not use this function for data exchange as it can produce
-not strict utf8 $octets! For strictly valid UTF-8 output use
-C<$octets = encode("UTF-8", $string)>.
-
 =head3 decode_utf8
 
   $string = decode_utf8($octets [, CHECK]);
 
+B<WARNING>: L</UTF-8 vs. utf8 vs. UTF8|This function accepts invalid UTF-8!>
```

Comment:
Also need to fix this hyperlink.

Reply:
fixed in https://github.com/dankogai/p5-encode/commit/67f59dd4092c12043182e427dfaa5abd5c791e58